corehq/apps/accounting/tests/test_autopay_with_api.py

```diff
@@ -20,13 +20,19 @@
 from corehq.apps.accounting.tests import generator
 from corehq.apps.accounting.tests.test_invoicing import BaseInvoiceTestCase
 from django.db import transaction
+from unittest import SkipTest
+from django.conf import settings
 
 
 class TestBillingAutoPay(BaseInvoiceTestCase):
 
     @classmethod
     def setUpClass(cls):
         super(TestBillingAutoPay, cls).setUpClass()
+        # Dependabot-created PRs do not have access to secrets.
+        # We skip test so the tests do not fail when dependabot creates new PR for dependency upgrades.
```

Comment:
Another reason for this is so they are skipped for developers running tests locally if they do not have an API key in their localsettings.

Reply:
Thank you! Updated here: 693afa1976b82b688a79dddcdb62760966e58149